docs/cncf_conformance.md

```diff
@@ -0,0 +1,167 @@
+# CNCF Conformance
+
+## Pre-Requisites
+
+To follow these instructions you should meet the following pre-requisites:
+
+* A Linux or MacOS machine with
+  [Terraform 0.11.x](https://www.terraform.io/downloads.html) installed.
+* A [Google Cloud](docs/gcp.md), [AWS](docs/aws.md) or [Azure](docs/azure.md)
+  account with enough permissions to provide the needed infrastructure
+
+## Preparing the infrastructure
+
+**NOTE:** These instructions are targeted at a Google Cloud Platform deployment.
+
+Start by cloning this repo:
+
+```
+$ git clone git@github.com:mesosphere/dcos-kubernetes-quickstart.git
+$ cd dcos-kubernetes-quickstart
+```
+
+Then generate the default infrastructure configuration:
+
+```
+$ make gcp
+```
+
+This will output sane defaults to `.deploy/desired_cluster_profile`. Now, edit
+said file and set the `gcp_project` and the `gce_ssh_pub_key_file` variables.
+Please, do not set a smaller instance (VM) type on the risk of failing to
```

Comment:
Please do not set a smaller instance (VM) type, or the Kubernetes deployment could fail.